src/lib/y2partitioner/dialogs/btrfs_subvolume.rb

```diff
@@ -178,18 +178,19 @@ def hierarchy_validation
           false
         end
 
-        # Updates #value by prefixing path with default subvolume path if it is necessary
+        # Updates #value by adding the subvolumes prefix
         #
         # Path should be a relative path. Starting slashes are removed. A popup message is
-        # presented when the default subvolume path is going to be added.
+        # presented when the subvolumes prefix is going to be added.
         #
+        # @see Y2Storage::Filesystems::Btrfs#subvolumes_prefix
         # @see Y2Storage::Filesystems::Btrfs#btrfs_subvolume_path
         def fix_path
           self.value = filesystem.canonical_subvolume_name(value)
           return if value.empty?
 
-          default_subvolume_path = filesystem.default_btrfs_subvolume.path
-          prefix = default_subvolume_path.empty? ? "" : default_subvolume_path + "/"
+          prefix = filesystem.subvolumes_prefix
+          prefix << "/" unless prefix.empty?
```

Comment:
as there was problem, is possible to add logging like what is value and what is prefix?